### DISCUSSION TITLE
blog/deploying-aspnet-core-web-api-to-aws-elastic-beanstalk-using-aws-codepipeline/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#45] [General]
# blog/deploying-aspnet-core-web-api-to-aws-elastic-beanstalk-using-aws-codepipeline/

In this article, we are going to be deploying ASP.NET Core Web API to AWS Elastic Beanstalk using AWS CodePipeline. In the process, we will learn the basics of AWS Codepipeline, AWS CodeBuild, and AWS Elastic Beanstalk.

https://codewithmukesh.com/blog/deploying-aspnet-core-web-api-to-aws-elastic-beanstalk-using-aws-codepipeline/

**Comment by JulioTrujilloLeon** [comment]
Hi Mukesh!
Your post has been really useful for me.
Wanted to update the process a little bit because I suppose things changed from the publish date on AWS platform.
First of all, I worked for me only copying and pasting the buildspec.yaml content into the AWS Codebuild wizard. The original way to take buildspec.yaml not worked for me.
As well as I made some changes into the build script doing something like this:
version: 0.2
phases:
  install:
    runtime-versions:
      dotnet: 6.0
  pre_build:
    commands:
      - dotnet restore AWSCodePipelineDemo.csproj
  build:
    commands:
      - dotnet build AWSCodePipelineDemo.csproj
  post_build:
    commands:
      - dotnet publish AWSCodePipelineDemo.csproj -c Release -o ./app
artifacts:
  files:
    - '**/*'
Hope this helps to new readers (from sept 2024 to ahead)

Bests, Julio
